v2/attackprotectionsuite/backend-setup.mdx

```diff
@@ -531,7 +533,7 @@ SuperTokens.init({
                             const bruteForceConfig = getBruteForceConfig(email, ip, actionType);
 
                             // we check the anomaly detection service before calling the original implementation of signIn
-                            let securityCheckResponse = await handleSecurityChecks({ ...input, requestId, email, bruteForceConfig, actionType });
+                            let securityCheckResponse = await handleSecurityChecks({ requestId, email, bruteForceConfig, actionType });
```

Comment:
u need to pass in password here?

Reply:
for sign in ? the password is already set. I can add the check if you want

Reply:
No i mean in the sign up api above